Util/src/main/java/io/deephaven/util/SafeCloseableList.java

```diff
@@ -56,23 +56,22 @@ public final void clear() {
 
     @Override
     public final void close() {
-        for (final SafeCloseable closeable : list) {
-            if (closeable != null) {
-                closeable.close();
-            }
+        try {
+            SafeCloseable.closeAll(list.iterator());
+        } finally {
+            list.clear();
```

Comment:
Should we extend `Closeable`, given this is a form of idempotency?

Reply:
The issue with that is simply `Closeable#close` has `throws Exception` on the method signature; feels like `SafeCloseable` is still appropriate to save users of `SafeCloseableList` from being required to catch `Exception`.